cve_bin_tool/checkers/perl.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+
+"""
+CVE checker for Perl
+References:
+https://www.cvedetails.com/vulnerability-list/vendor_id-1885/Perl.html
+"""
+from ..util import regex_find
+
+
+def get_version(lines, filename):
+    """
+    returns version information for perl as found in a given file.
+    """
```

Comment:
You're missing the VPkg: string as described here: https://github.com/intel/cve-bin-tool/blob/master/cve_bin_tool/checkers/README.md

It should be VPkg: perl, perl